nf_core/modules/lint/main_nf.py

```diff
@@ -196,7 +196,7 @@ def check_process_section(self, lines):
     correct_process_labels = ["process_low", "process_medium", "process_high", "process_long"]
     process_label = [l for l in lines if "label" in l]
     if len(process_label) > 0:
-        process_label = process_label[0].split()[1].strip().strip("'").strip('"')
+        process_label = re.search("process_[A-Za-z]*", process_label[0]).group(0)
```

Comment:
We want at least one character matching..

```suggestion
        process_label = re.search("process_[A-Za-z]+", process_label[0]).group(0)
```